GNUmakefile

```diff
@@ -601,6 +602,7 @@ distrib: all
 	$(MAKE) -C utils/afl_network_proxy
 	$(MAKE) -C utils/socket_fuzzing
 	$(MAKE) -C utils/argv_fuzzing
+	$(MAKE) -C utils/plot_ui
```

Comment:
a) MUST have a `-` in front: `-$(MAKE)` is the build can fail if the packages are not installed and the the whole afl++ build fails.
b) I am not sure it should always be built. I would propose to first put it in for people having to manual build it if they want to for the dev branch and then asking on the fuzzing discord channel who useful people find it, and if the majority find it useful then another PR that always builds it.

Reply:
Thank you for informing about point *a)*.

I am sorry, but I am unable to understand what you're trying to say in point *b)*. Should I create a new make target in which `afl-plot-ui` will be built? I have opened the PR for the `AFLplusplus:dev` branch, so I don't suppose this will be merged in the stable branch. I am not able to see what exactly I should be changing in the Makefile.

Reply:
I am basically saying, move this from
```
$(MAKE) -C utils/plot_ui
```
to
```
#-$(MAKE) -C utils/plot_ui
```

Reply:
Okay, got it. I have now commented out all the lines where I was building `utils/plot_ui`.